src/Player.cpp

```diff
@@ -67,3 +68,25 @@ void Player::ResetHit()
 {
 	gequacked = false;
 }
+
+void Player::ScreenBounds()
```

Comment:
So yeah, if you would move that function call after collision and listener position update, it will also need to correct them as well 🤔 We will need to modularize this code soon, I think @Ania000 in #24 has already suggested one approach to such function (have fun resolving code conflicts xD)